src/order/compactly_generated.lean

```diff
@@ -420,12 +420,15 @@ instance is_atomistic_of_is_complemented [is_complemented α] : is_atomistic α
     exact le_Sup ⟨ha.of_is_atom_coe_Iic, a.2⟩ }
 end, λ _, and.left⟩⟩
 
-/-- See Theorem 6.6, Călugăreanu -/
-theorem is_complemented_of_Sup_atoms_eq_top (h : Sup {a : α | is_atom a} = ⊤) : is_complemented α :=
-⟨λ b, begin
+/-- This encapsulates some of the general information one can get out of an atomistic lattice.
+For more specific data, e.g. the existence of a maximal independent set of atoms, see below. -/
```

Comment:
I would prefer a more informative docstring. Something like
```suggestion
/-- In an atomic lattice, every element `b` has a complement of the form
`Sup s`, where each element of `s` is an atom. See also `is_complemented_of_Sup_atoms_eq_top`. -/
```

Reply:
BTW, a module section docstring with something like "Now we are going to prove that an atomic lattice is a complemented lattice. We start with the most explicit version of the statement, see `...`: (explanation). Then we use deduce two corollaries from this fact: ... and ...